libpod/runtime_img.go

```diff
@@ -1037,6 +1020,12 @@ func (r *Runtime) GetHistory(image string) ([]ociv1.History, []types.BlobInfo, s
 	return oci.History, src.LayerInfos(), img.ID, nil
 }
 
+//Import imports an oci format image archive into storage as an image
+func (r *Runtime) Import(path, reference string, writer io.Writer, signingOptions image.SigningOptions, imageConfig ociv1.Image) (*image.Image, error) {
+	return image.Import(path, reference, writer, signingOptions, imageConfig, r.imageRuntime)
+}
+
+/*
 // ImportImage imports an OCI format image archive into storage as an image
 func (r *Runtime) ImportImage(path string, options CopyOptions) (*storage.Image, error) {
```

Comment:
Can all these commented-out functions be safely removed?

Reply:
sure!